resources/elasticache-replicationgroups.go

```diff
@@ -0,0 +1,54 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elasticache"
+)
+
+type ElasticacheReplicationGroup struct {
+	svc     *elasticache.ElastiCache
+	groupID *string
+}
+
+func init() {
+	register("ElasticacheReplicationGroup", ListElasticacheReplicationGroups)
+}
+
+func ListElasticacheReplicationGroups(sess *session.Session) ([]Resource, error) {
+	svc := elasticache.New(sess)
+
+	params := &elasticache.DescribeReplicationGroupsInput{MaxRecords: aws.Int64(100)}
```

Comment:
We should check if there are additional pages. See https://github.com/rebuy-de/aws-nuke/blob/592a342a1848d062ee4fd6118e0089a138c7d2af/resources/databasemigrationservice-certificates.go#L39-L43 for an example.

Reply:
@svenwltr apologies for the incredibly delayed response. I've added support for pagination. Would you mind reviewing this again?